sdk/helper/testcluster/docker/environment.go

```diff
@@ -732,7 +732,10 @@ func (n *dockerClusterNode) start(ctx context.Context, caDir string, opts *Docke
 	n.container = svc.Container
 	netName := opts.NetworkName
 	if netName == "" {
-		netName = "bridge"
+		if len(svc.Container.NetworkSettings.Networks) > 1 {
+			return fmt.Errorf("Set d.RunOptions.NetworkName instead for container with multiple networks: %v", svc.Container.NetworkSettings.Networks)
+		}
+		for netName, _ = range svc.Container.NetworkSettings.Networks {}
```

Comment:
Why the empty loop?

Reply:
Sadly this is a map... Is there an easier way to get the (only) key in a map?

Reply:
Ah, I figured that was why you needed the other loop, but didn't notice that there was an existing variable used here.  Thanks for adding the comment.